workflows/hifi2genome.nf

```diff
@@ -46,6 +40,23 @@ for (param in checkPathParamList) { if (param) { file(param, checkIfExists: true
 */
 
 workflow HIFI2GENOME {
+    ch_versions = Channel.empty()
+
+    // read in samplesheet
+    ch_reads = INPUT_CHECK(ch_input)
+    ch_versions = ch_versions.mix(INPUT_CHECK.out.versions)
+
+    // assembly with flye
+    FLYE (
+        ch_reads.reads, //had to explicitly define .reads here even though module says it wants tuple(meta), path(reads)
+        "--pacbio-hifi"
+    )
+    ch_versions = ch_versions.mix(FLYE.out.versions)
+
+    // quality check assembly with busco
```

Comment:
The comments are perfect, but I'd remove `BUSCO` as an import from here, until you're ready to actually implement it.